src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserFactory.java

```diff
@@ -0,0 +1,53 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import lombok.Builder;
+import lombok.Getter;
+
+public final class ForgottenPasswordUserFactory {
+
+  @Builder
+  public static class ForgottenPasswordUser {
+    private User user;
+
+    @Getter
+    private String email;
+
+    @Getter
+    private String emailPassword;
+
+    public String getUsername() {
+      return user.getUserName();
+    }
+
+    public String getPassword() {
+      return user.getPassword();
+    }
+  }
+
+  private static Credentials credentials = Configuration.getCredentials();
+
+  private static ForgottenPasswordUser user(User user, String email, String password) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "user" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)